instrumentor/controllers/agentenabled/pods_webhook.go

```diff
@@ -224,13 +247,14 @@ func (p *PodsWebhook) injectOdigosInstrumentation(ctx context.Context, pod *core
 	return nil
 }
 
-func (p *PodsWebhook) injectOdigosToContainer(containerConfig *odigosv1.ContainerAgentConfig, podContainerSpec *corev1.Container, pw k8sconsts.PodWorkload, serviceName string, config common.OdigosConfiguration) (bool, error) {
+func (p *PodsWebhook) injectOdigosToContainer(containerConfig *odigosv1.ContainerAgentConfig, podContainerSpec *corev1.Container,
+	pw k8sconsts.PodWorkload, serviceName string, config common.OdigosConfiguration) (bool, map[string]struct{}, error) {
```

Comment:
consider removing the first returned value `volumeMounted` as it now redundant (if the map is not empty it means a volume is mounted)